site/app/models/gradeable/Gradeable.php

```diff
@@ -2358,6 +2358,7 @@ public function setAllowedMinutesOverrides(array $overrides): void {
      *
      * @param string $user_id
      * @return bool
+     *
```

Comment:
Same here.  This change is unrelated to the rest of this PR.  Whitespace changes like this can unintentionally cause merge conflicts in other PRs.